plan_delegator/src/plan_delegator.cpp

```diff
@@ -183,6 +198,7 @@ namespace plan_delegator
                 break;
             }
         }
+        latest_trajectory_plan.initial_longitudinal_velocity = std::max(latest_twist_.twist.linear.x, 2.2352); // TODO make config paramete or evaluate if this max call needed? Would 0 cause an issue?
```

Comment:
I think per our discussion 0 would cause some issues with starting up from a standstill. I'd be in favor of making this a config param though, probably a global one. We should just define a minimum "crawl speed" and anything below that should only occur on the way to 0.